### DISCUSSION TITLE
rmview having errors with libcrypto.so.1.0.2

### DISCUSSION THREAD

**Discussion by spiffyfrodo** [bordaigorl/rmview#67] [Q&A]
As the title suggest. I recently opened up my tablet after months of not using it. I'm an teacher by trade and love the rmview program. I recently upgraded to a rM2 in December and by January 2021 had it running with your new code. I used it until February then we went back to onsite teaching. Now we are back to online teaching and it produces an error in terminal.

:::::::::::::::::::
/home/root/rM-vnc-server-standalone: error while loading shared libraries: libcrypto.so.1.0.2: cannot open shared object file: No such file or directory.
:::::::::::::::::::

I am also getting this GUI popup when I try to run rmview:

:::::::::::::::::::
Please check your reMarkable is properly configured, see the documentation for instructions.

Connection failed: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionRefusedError'>: Connection was refused by other side: 111: Connection refused.
]
:::::::::::::::::::

I did install a new router since starting up the remarkable 2 and thought it was a firewall issue. But I put the old router in and tried it on both Ubuntu 20.04 and my old laptop 18.04 without success (same error)

**Comment by bordaigorl** [comment]
Looks like you updated to 2.6 and haven't seen issue #57 
Try and follow the instructions there, it should work if you copy the libcrypto file.

**Reply by spiffyfrodo** [reply]
Many thanks. When i saw that post I was still in the mindframe that it was a firewall issue. Sorry for the duplicate. :(
